plugins/woocommerce/client/legacy/js/frontend/checkout.js

```diff
@@ -550,7 +556,14 @@ jQuery( function( $ ) {
 
 							// Add new errors
 							if ( result.messages ) {
-								wc_checkout_form.submit_error( result.messages );
+								var $msgs = $( result.messages )
+									.removeAttr( 'role' )
+									.attr( 'tabindex', '-1' );
```

Comment:
Overall, `<ul>` elements must have the default `list` role (`role="list"`). This became a problem in this task because the list items have been wrapped by the `<a>` tag and Voice Over keeps announcing the content infinitely.
The solution was to remove the `role="alert"` attribute from the `<ul>` element and add it to the parent.